### ISSUE TITLE
Can it be used to separate music?

### ISSUE THREAD

**Issue by chenjunbin666** [d-kitamura/AuxIVA-ISS#3]
Hello, thank you very much for sharing. I saw that there are two music signals in your input folder. As far as I know, its objective function is based on speech statistical models such as Laplacian distribution. So, is it applicable to music signals ?

**Comment by d-kitamura** [comment]
Yes, the source generative model of this script is the multivariate Laplace distribution, which is proposed in the original IVA paper (Taesu Kim et al., 2007) and is often assumed for speech sources. As you said, the Laplace distribution may be not suitable for other sources, such as white noise, babble noise, and some musical instruments. If the separation results for such sources are not satisfactory, you may improve them by replacing the source generative model, although the update rule must be derived with the new model. 
In my experience, IVA with the Laplace distribution can somewhat separate music mixtures to some extent. But it may be improved by using the zero-mean complex Gaussian distribution with time-frequency-varying variance (often called "local Gaussian model"), which is used in interdependent low-rank matrix analysis (ILRMA).
